.travis.yml

```diff
@@ -0,0 +1,5 @@
+language: objective-c
+osx_image: xcode7.3
```

Comment:
Xcode 8 is Swift 3, which hasn't been merged in yet 😁 I can change it after the other PR is approved